### PR TITLE
[shockbs.json] Pls Merge This When _domainkey is supported

### PR DESCRIPTION
**Pull request by shockbs**
## Requirements
Unless explicitly specified otherwise by a **maintainer** or in the requirement description, your domain must pass **ALL** the indicated requirements above.

Please note that we reserve the rights not to accept any domain at our own discretion.

- [x] The file is in the `domains` folder and is in the JSON format.
- [x] The file's name is all lowercased and alphanumeric. 
- [x] You have completed your website. 
- [x] The website is reachable.  
- [x] You're not using Vercel.  
- [x] The CNAME record doesn't contain `https://` or `/`.  
- [x] There is sufficient information at the `owner` field.  

## Website Link/Preview

 thank you, this is for mail

### CODE REVIEW

File: domains/shockbs.json [hunk 1]
```diff
@@ -4,6 +4,8 @@
     "email": "apipherng@gmail.com"
   },
   "record": {
-    "CNAME": "shockbshdhhdjdjndjd.onrender.com"
+    "CNAME": "shockbshdhhdjdjndjd.onrender.com",
+    "MX": ["mxa.mailgun.org","mxb.mailgun.org"],
+    "TXT": "v=spf1 include:mailgun.org ~all"
```
Comment (CuteDog5695) [inline domains/shockbs.json:9]: You can't use CNAME with other records

**Review by iostpa — DISMISSED** [review]

File: domains/shockbs.json (hunk 1 above)
Comment (shockbs) [reply domains/shockbs.json]: i've changed to A records, thanks!

File: domains/mx._domainkey.shockbs.json
Comment (DEV-DIBSTER) [inline domains/mx._domainkey.shockbs.json:1]: I believe we do not support this kid of record.

**Review by DEV-DIBSTER — DISMISSED** [review]

File: domains/mx._domainkey.shockbs.json
Comment (shockbs) [reply domains/mx._domainkey.shockbs.json:1]: pls support dkim

**Review by wdhdev — APPROVED** [review]

**Pull request merged**